packages/components/src/DataTable/DataTable.module.css

```
@@ -104,7 +104,7 @@
 
 .clickableRow:hover,
 .clickableRow:hover :first-child {
-  background: var(--color-surface--background);
+  background: var(--color-interactive--background);
```

A row in this context should be considered a container
```suggestion
  background: var(--color-surface--background);
```